### DISCUSSION TITLE
Problem in sample in Get Started tutorial

### DISCUSSION THREAD

**Discussion by BlowyXDU** [iTwin/itwinjs-core#4531] [Q&A] [answered]
Hi!I just followed the steps in sample in Get Started,everything was fine but the web didn't show anything,just black screen,what happened?Thanks for answering me :).Here are the result and errors report.
![图片](https://user-images.githubusercontent.com/116451744/197350389-3f26dba6-3990-4a7a-a05c-8c390cbfa433.png)
![图片](https://user-images.githubusercontent.com/116451744/197350432-030fd0ab-cb8a-460b-abe7-2c31bff2a996.png)
![图片](https://user-images.githubusercontent.com/116451744/197350449-96262e5f-ace5-4343-8ff1-598000bbdfbe.png)

**Comment by pmconne** [comment, marked as answer]
Hi @BlowyXDU,

I just followed [the tutorial](https://developer.bentley.com/tutorials/web-application-quick-start/) and observed the same result.
Those errors about "404 (the specified blob does not exist.)" are expected, and they indicate that the app opened your iModel and began (attempting to) download tiles to display it.

We will need to investigate.

**Reply by pmconne** [reply]
Tracking in #4532. Thanks for reporting the problem @BlowyXDU.

**Reply by BlowyXDU** [reply]
Thanks for answering me,i will track this issue :).

**Comment by aruniverse** [comment]
Hi,

This has been fixed in iTwin.js 3.4.2, please upgrade your frontend to consume this version. If issues persist or you run into any other issues please let us know.
